packages/frontend/web/browser.ts

```diff
@@ -28,10 +29,7 @@ const go = () => {
 
         hydrateApp(React.createElement(Article, { data }), container);
     }
-
-    import('@frontend/web/lib/ga').then(({ init }) => {
-        init();
-    });
+    initGa();
```

Comment:
@GHaberis just pointed out IRL that this has to execute before we do any other GA stuff. Maybe we should move this higher up the module.

Reply:
I've split the GA module into two functions:

- initialise the cold queue and tracker
- send initial page view

The former executes before the app mounts, the latter executes after.